apps/dashboard/.env.example

```diff
@@ -8,3 +8,4 @@ VITE_NOVU_APP_ID=
 VITE_INTERCOM_APP_ID=
 VITE_GTM=
 VITE_SELF_HOSTED=
+VITE_EMAIL_DELAY_DIGEST_ENABLED=
```

Comment:
are we not using the launchdarkly for FF in new dashboard?

Reply:
ok i see, we will be using launchdarkly
here is the [PR](https://github.com/novuhq/novu/pull/7133/files) to add that.
You will need to update it after that PR merges